examples/src/content/jcr_root/content/core-components-examples/library/.content.xml

```diff
@@ -352,6 +352,27 @@
                     sling:resourceType="core/wcm/components/text/v2/text"
                     text="&lt;p>No components found.&lt;/p>&#xa;"
                     textIsRich="true"/>
+                <teaser_1138854624
```

Comment:
Can we add this at its correct position in the list, components are listed alphabetically.